concept-amendments.html

```diff
@@ -341,6 +359,31 @@ <h2 id="cryptoconditions">CryptoConditions</h2>
 </tbody>
 </table>
 <p>Although this amendment is enabled, it has no effect unless the <a href="#suspay">SusPay</a> amendment is also enabled. Ripple does not expect SusPay to become enabled. Instead, Ripple plans to incorporate crypto-conditions in the <a href="#escrow">Escrow</a> amendment.</p>
+<h2 id="enforceinvariants">EnforceInvariants</h2>
+<table>
+<thead>
+<tr>
+<th align="left">Amendment ID</th>
+<th align="left">Status</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td align="left">DC9CA96AEA1DCF83E527D1AFC916EFAF5D27388ECA4060A88817C1238CAEE0BF</td>
+<td align="left">In voting; expected 2017-06-29</td>
+</tr>
+</tbody>
+</table>
+<p>Adds sanity checks to transaction processing to ensure that certain conditions are always met. This provides an extra layer of protection against bugs in transaction processing that could otherwise cause exploits and vulnerabilities in the Ripple Consensus Ledger. Ripple expects to add more invariant checks in future versions of <code>rippled</code> without additional amendments.</p>
+<p>Introduces two new transaction error codes, <code>tecINVARIANT_FAILED</code> and <code>tefINVARIANT_FAILED</code>. Changes transaction processing to add the new checks. Adds a configuration option to disable invariant-checking.</p>
```

Comment:
I think we need to remove: "Adds a configuration option to disable...".  This was originally a planned part of the feature, but was removed in the final implementation. As it stands now, you can't disable the checks without changing the code itself.